Automattic/jetpack
# Jetpack Google Fonts: Fix the fonts provided by theme are still added

Related to p1702367211316529/1702365341.631869-slack-CRWCHQGUB

## Proposed changes:


* We have to ignore the fonts that are provided by the theme to keep the behavior consistent between the Site Editor & frontend page

| Before | After |
| - | - |
| ![Screenshot 2023-12-13 at 3 40 29 PM](https://github.com/Automattic/jetpack/assets/13596067/bf936654-2610-451c-a5b5-e437983c4356) | ![Screenshot 2023-12-13 at 3 39 51 PM](https://github.com/Automattic/jetpack/assets/13596067/bf3f58ce-da9b-4822-bd2f-762eaea8b263) |

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this change to your WoA site via Jetpack Beta Tester
* Active the Adventurer theme
* Go to the Site Editor
* Open the Styles > Typography > Any Element (e.g.: Text) > Font
* Pick the Rubik font
* Pick different font weights
* Make sure that only `font-weight: 900` works
* Go to your frontend page
* Make sure that only `font-weight: 900` works as well
